CONTRIBUTING.md

```diff
@@ -34,7 +34,7 @@ $ mvn install
 Run:
 ```$sh
 $ cd exonum-java-binding-core/rust/ejb-app
-$ cargo install --debug --force
+$ cargo build
```

Comment:
Would you please add to the task on updating to 1.31 that this will need to be reverted back to `install`?